content/_data/sidebar.json

```diff
@@ -204,6 +204,10 @@
               "title": "Cypress App",
               "slug": "cypress-app"
             },
+            {
```

Comment:
I don't think I would consider this a Cypress Core Concept. It feels like this content being under References makes more sense.

Reply:
We moved it back here since this is where it was initially, before it was removed in v10. I am ok with either though. Maybe References does make more sense while Studio is experimental..

Reply:
References includes things like experiments(as Studio still is), advanced topics, bundled libs, etc. I'd say it feels more at home there vs. a fundamental concept of the product.